### DISCUSSION TITLE
Feature Request: Voice for WearOS on Smartwatches?

### DISCUSSION THREAD

**Discussion by jiang-qian** [PaulWoitaschek/Voice#1749] [Ideas]
### Overview

I have been a longtime happy user of Voice on phone. It is in fact my most used app on Android, since I listen to audiobooks  at least an hour a day while I'm cooking or working! My wife is also a heavy and happy user. Thank you indeed for this very useful app!

I recently got a WearOS watch (Samsung Galaxy Watch 4, running Wear OS 3), and it works great as a controller of audiobook playback on my phone. However, it is quite possible to listen to local music on wear OS, i.e. those music loaded to the watch  itself and can be played back without a phone. 

But not for local audiobooks! The music player on Wear OS are geared towards songs, each of them short, in a playlist, and only have buttons to skip forward or backward by one song. For audiobook, however, it is crucial to skip forward and backwards for a set time (e.g. 30 seconds) and have a usable progress bar to drag to a particular position in a long audiobook. I spent hours trying out WearOS audiobook and music apps one by one, and all of them are either lacking features for audiobook or incredibly buggy.

It would be wonderful if a very simple but elegant app like Voice could be ported to Wear OS! My understanding is that Wear OS is a heavily modified version of Android. Perhaps it is not such a herculean task to port Voice to it? 

There are more and more Wear OS watches now in the market, not just Samsung Galaxy 4 and 5 but many other brands, including Google itself, are making wearOS 3 watches. There are already nearly 10 million Galaxy wearOS 3 watches on the market.

In fact, I want to have such an app on WearOS so much that I'm not only willing to pay for it once it is out, but to sponsor a porting process as long as it is reasonable and within my means. Would the Voice developer be interested in looking into this?

Again, thank you indeed for such a wonderful app on Android!

### How to reproduce

Try to find Voice equivalent on Wear OS

### Version

most recent on on Android

### Android Version

Android 13

### Device Model

Samsung galaxy watch 4 Classic

**Comment by aleksfadini** [comment]
I know it's a big ask, but I intensely agree.
